applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -645,19 +645,19 @@ AT_MONITOR(at_notify, ANY, notification_handler);
 static void notification_handler(const char *notification)
 {
 	if (get_slm_mode() == SLM_AT_COMMAND_MODE) {
-		(void)slm_uart_tx_write(CRLF_STR, strlen(CRLF_STR), true);
-		(void)slm_uart_tx_write(notification, strlen(notification), true);
+		(void)slm_uart_tx_write(CRLF_STR, strlen(CRLF_STR), true, false);
```

Comment:
Could indicate for that one as well, but as the indications are supposed to come only for data and notifications, I thought this would be more in tune with that.

Reply:
Does not really matter, but changed.